60_networks.Rmd

```diff
@@ -0,0 +1,664 @@
+
+The "signed" method is used to transform associations into dissimilarities (argument `dissFunc`). Thus, strongly negatively associated taxa have a high dissimilarity and conversely a low similarity, corresponding to low edge weights on the network plot.
+
+```{r netConstruct_single, eval=FALSE}
+peerj_net <- netConstruct(peerj_phy,
+                          taxRank = "genus",
+                          filtTax = "highestFreq",
+                          filtTaxPar = list(highestFreq  = 50),
+                          filtSamp = "totalReads",
+                          filtSampPar = list(totalReads = 1000),
+                          measure = "spring",
+                          measurePar = list(nlambda=30, 
+                                            rep.num=30,
+                                            Rmethod = "approx"),
+                          normMethod = "none", 
+                          zeroMethod = "none",
+                          sparsMethod = "none", 
+                          dissFunc = "signed",
+                          seed = 13075
+                          verbose = 3)
+```
+
+
+```{r, eval=FALSE, echo=FALSE}
+saveRDS(peerj_net, file = "general/network_files/peerj_net.rds")
+```
+
+
+```{r, echo=FALSE}
+peerj_net <- readRDS("general/network_files/peerj_net.rds")
+```
+
+`netConstruct` returns an object of the class `microNet`, which contains all matrices generated during network construction. These are the basis for the network analysis.
+
+The object also contains an edge list, giving each edge's estimated association, dissimilarity, and adjacency. Let's take a quick look at the edges with the highest and lowest edge weights:
+
+```{r}
+edgelist <- peerj_net$edgelist1[order(peerj_net$edgelist1$adja, decreasing = TRUE), ]
+head(edgelist)
+tail(edgelist)
+```
+
+### Network analysis
+
+The `microNet` object is now passed to `netAnalyze` to perform network analysis.
+
+The function computes several common network analysis measures such as centrality measures, cluster assignment, the graphlet correlation matrix, as well as global network measures like the average path length and the clustering coefficient.
+
+Here, `centrLCC` is set to `TRUE` meaning that centralities are calculated only 
+for nodes in the largest connected component (LCC). 
+
+Clusters are identified using greedy modularity optimization 
+(by `cluster_fast_greedy` from [`igraph`](https://igraph.org/r/) package).
+
+Hubs are nodes with an eigenvector centrality value above the empirical 
+95% quantile of all eigenvector centralities in the network (argument `hubPar`).
+
+`weightDeg` and `normDeg` are set to `FALSE` so that the degree of a node is
+simply the number of its adjacent nodes.
+
+By default, a heatmap of the Graphlet Correlation Matrix (GCM) is returned (with 
+graphlet correlations in the upper triangle and significance codes resulting 
+from Student's t-test in the lower triangle). 
+See `?calcGCM` and `?testGCM` for details.
```

Comment:
Same with these options - avoid replicating argument details that are easily found from the package documentation itself. Instead of explaining each argument, consider showing some illustrative complementary examples on what is possible to do with these different options.